tests/phpunit/tests/Tracking.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Tests;
+
+use Spy_REST_Server;
+use WP_REST_Request;
+
+class Tracking extends \WP_UnitTestCase {
+	protected static $user_id;
+
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$user_id = $factory->user->create(
+			[
+				'role' => 'administrator',
+			]
+		);
+	}
+
+	public static function wpTearDownAfterClass() {
+		self::delete_user( self::$user_id );
+	}
+
+	public function test_add_optin_field_to_rest_api() {
+		wp_set_current_user( self::$user_id );
+		( new \Google\Web_Stories\Tracking() )->init();
+		add_user_meta( get_current_user_id(), \Google\Web_Stories\Tracking::OPTIN_META_KEY, true );
+
+		$request  = new WP_REST_Request( 'GET', sprintf( '/wp/v2/users/%d', self::$user_id ) );
+		$response = rest_get_server()->dispatch( $request );
+		$data     = $response->get_data();
+		$this->assertArrayHasKey( \Google\Web_Stories\Tracking::OPTIN_META_KEY, $data['meta'] );
+		$this->assertTrue( $data['meta'][ \Google\Web_Stories\Tracking::OPTIN_META_KEY ] );
+	}
+
+	public function test_register_tracking_script() {
+		$this->assertArrayHasKey( \Google\Web_Stories\Tracking::SCRIPT_HANDLE, wp_scripts()->registered );
+		$this->assertFalse( wp_scripts()->registered[ \Google\Web_Stories\Tracking::SCRIPT_HANDLE ]->src );
```

Comment:
```suggestion
		$this->assertTrue(  wp_script_is( \Google\Web_Stories\Tracking::SCRIPT_HANDLE ], 'registered' );
```

Reply:
That's not the same. We need both.

1. Verify that script is registered
1. Verify that script `src` is false